IOfoundation/launchpad-client
# CS-167 Access data from API and display org names

** Requires branch CS-134 merged into Ohana-API master
# To run locally:
1. Pull latest from fullstacklabs/Ohana-API
2. run `cd app/job
      ruby parse_csv_job.rb`
3. Should see 8 new CSVs in /data --> setup db & run `script/import` & `bin/rake create_categories`
4. start api
5. start client
6. go to `http://localhost:3000/businesses`
